### DISCUSSION TITLE
rClone fork stores large movie files on the hard drive

### DISCUSSION THREAD

**Discussion by pzinser** [itsToggle/plex_debrid#434] [General]
I was poking around the folders involved with rclone and found the following folder, C:\Users\"username"\AppData\Local\rclone\vfs\remoterealdebrid\movies.

Now, I found this when the "virtual drive" of real debrid was NOT mounted.  All of the movie files from my real debrid account are present here in their full size.  They are NOT playable, but the fact they're taking up so much of my hard drive space seems at odds with the purpose of "streaming" them from real debrid.  After all, one could just download the "movie proper" from real debrid and "stream" it locally on your home network.  

OR, am I missing something?  I do like this programming very much.    Thanks.

**Comment by itsToggle** [comment]
You are using vfs-cache-mode, which is supposed to do what you just described. You can read about is purpose in the official rclone documentation. If you were to use the recommended mount command, nothing would be stored locally.

**Reply by pzinser** [reply]
Thank you.

**Comment by berot3** [comment]
What was the mount command you used? I'm just curious :)

**Reply by pzinser** [reply]
./rclone.exe cmount realdebrid: X: --vfs-cache-mode writes

I used the above mount command instead of the one in the directions because I got the following error message upon launching the mount command:

NOTICE: realdebrid root '': --vfs-cache-mode writes or full is recommended for this remote as it can't stream
The service rclone has been started.

SO, I thought I HAD NO CHOICE as I WANTED to stream,  so I made the change and continued onward, completely unaware for a long time that it was storing copies of the files on my hard drive.  
Once I realized this, I changed the mount command back to what was in the directions.  NOW, I STILL get the same error message every time I launch the command and mount rclone, BUT the streaming works just fine.

I've not pressed the issue, but it begs the question, WHY do I get that error message??  It says "it can't stream", but it DOES.

**Reply by itsToggle** [reply]
That's not an error, it's a "notice" from rclone. You can read about VFS cache mode in the official rclone documentation.

**Reply by berot3** [reply]
> In this context streaming means uploading a file which rclone doesn't know the size of in advance.
> 
> You can't do that with (most) webdav servers.

https://forum.rclone.org/t/webdav-vfs-cache-mode-writes-or-full-is-recommended-for-this-remote-as-it-cant-stream/34841/2

But I think this is only relevant to files and zips, but not media. So can be ignored since media can be streamed anyways. 

Understanding all of this might help 😄 https://rclone.org/commands/rclone_mount/
It's **A LOT**! But we should be fine with the defaults.
